client/lib/track-form/index.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
For consistency, should the extension of this file be `.jsx` since it includes JSX?